modules/ocs2_api/src/main/scala/gem/ocs2/pio/PioParse.scala

```diff
@@ -1,4 +1,4 @@
-// Copyright (c) 2016-2021 Association of Universities for Research in Astronomy, Inc. (AURA)
+// Copyright (c) 2016-2022 Association of Universities for Research in Astronomy, Inc. (AURA)
 // For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
 
 package gem.ocs2.pio
```

Comment:
I suppose all ocs2 classes will disappear soon

Reply:
Yes, that's the idea.